src/components/ec-input-field/ec-input-field.vue

```diff
@@ -127,6 +130,7 @@ const props = withDefaults(defineProps<InputFieldProps>(), {
   type: InputFieldType.TEXT,
   label: '',
   labelTooltip: '',
+  inputTooltip: '',
```

Comment:
why do we need this prop? can't we use modelValue?

Reply:
we can, but I added it in case we don't want to show a tooltip. Do you think it's better then to have a show-input-tooltip prop instead?

Reply:
I think we should always show it if the content doesn't fit.

Reply:
I thought it would be a bit weird to have a tooltip for an input when the user is just typing and can simply scroll

Reply:
Ok

Reply:
BTW if tooltip is passed, then we should show it in any case, so `getInputTooltipOptions` is not needed.